django_test_migrations/autonames.py

```diff
@@ -17,7 +17,10 @@
 _IGNORE_APP_MIGRATIONS_SPECIAL_KEY: Final = '*'
 
 
-def check_migration_names(*args, **kwargs) -> List[CheckMessage]:
+def check_migration_names(  # noqa: WPS210
```

Comment:
We also tend not to ignore complexity violations. Let's make a new function instead!

This will help us to keep the code simple in the long run.